<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowInfo</class>
 <widget class="QDialog" name="ShowInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushLookButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>341</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushWatchMarksButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр по оценкам</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushWatchSubjectButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр по предметам</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineMarksEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineSubjectEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>440</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushWatchIDButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Просмотр по ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIDEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>470</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
